cdb2api/cdb2api.c

```diff
@@ -5985,18 +6059,11 @@ static int cdb2_run_statement_typed_int(cdb2_hndl_tp *hndl, const char *sql, int
         cdb2_get_dbhosts(hndl);
     }
 
-    int tmsec = 0;
-
-    // Add wait if we have already tried on all the nodes.
-    if (!hndl->sb && (retries_done > hndl->num_hosts)) {
-        tmsec = (retries_done - hndl->num_hosts) * 100;
```

Comment:
Still need the similar logic on line 6026 6093? (scroll down to int tmsec = (retries_done - hndl->num_hosts) * 100;). Uses 1000 not max_time

Reply:
this tmsec wasn't used anywhere so just removed it, it was being recalculated and used here
https://github.com/bloomberg/comdb2/pull/5587/files#diff-6394e8e4ae4483e61b99bde41b99db08004741ecdf0242162232abbf5386561cR6093